test-suite/Makefile

```diff
@@ -102,7 +102,7 @@ REDIR := $(if $(VERBOSE),,> /dev/null 2>&1)
 
 # read out an emacs config and look for coq-prog-args; if such exists, return it
 get_coq_prog_args_helper = sed -n s'/^.*coq-prog-args:[[:space:]]*(\([^)]*\)).*/\1/p' $(1)
-get_coq_prog_args = $(strip $(shell $(call get_coq_prog_args_helper,$(1))))
+get_coq_prog_args = $(strip $(shell $(call get_coq_prog_args_helper,$(1)))) -w -deprecated-hint-without-locality
```

Comment:
I'd put it at the start not at the end

Reply:
also get_coq_prog_args_in_parens below needs to use a version without this if we don't want to spam